node/helper-module/serverless.yml

```diff
@@ -0,0 +1,10 @@
+service: example-tracing-as-module-can-delete
```

Comment:
`example-tracing-as-module-can-delete`? perhaps something referencing the helper module?